website/content/en/docs/advanced-topics/heterogeneous.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Heterogeneous Cluster Support
+linkTitle: Heterogeneous Cluster Support
+weight: 80
+---
+
+An operator deployed to a heterogeneous cluster may not support all of the node architectures in the cluster. This has scheduling implications for both the operator as well as its operands.
+
+### Setting Operator Default Architectures
```

Comment:
I think we need to add the info about the CSV at the end.
Why?
I can have an Operator built with the tool and I should to the config for All Pods/Deployments. Therefore, integrating with OLM to publish in the catalogs is Optional. So, IMHO it would be better at the end with something like Now, if you integrate your project with OLM then ...

Reply:
Got it - so we want to treat the CSV bit as a special case of the overall story of setting podAffinity for your pods, not as separate step entirely (since it's ultimately optional).

Reply:
Separated this out into its own section.